### DISCUSSION TITLE
Problem with HA

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#809] [Ask A Question] [answered]
Hi!

I am really noob for this but trying to test this with Intel Nuc where I have Ubuntu 20.04 and running Home Assistant with docker.

I've installed FrigateNVR as an HA addon and starting with cleaniest frigate.yml conf:

`mqtt:
  host: 192.168.1.60
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: xxx
  password: xxx

cameras:
  front:
    ffmpeg:
      inputs:
        - path: rtsp://admin:password@192.168.1.69:554/Streaming/Channels/1
          roles:
            - detect
            - rtmp
    width: 1280
    height: 720
    fps: 5`

When starting addon I'll get these errors in log:

`* Starting nginx nginx
   ...done.
frigate.app                    INFO    : Creating directory: /tmp/cache
Starting migrations
peewee_migrate                 INFO    : Starting migrations
There is nothing to migrate
peewee_migrate                 INFO    : There is nothing to migrate
frigate.app                    INFO    : Camera processor started for front: 40
frigate.app                    INFO    : Capture process started for front: 41
detector.coral                 INFO    : Starting detection process: 37
frigate.edgetpu                INFO    : Attempting to load TPU as usb
frigate.edgetpu                INFO    : No EdgeTPU detected.
Process detector:coral:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 152, in load_delegate
    delegate = Delegate(library, options)
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 111, in __init__
    raise ValueError(capture.message)
ValueError
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/frigate/frigate/edgetpu.py", line 124, in run_detector
    object_detector = LocalObjectDetector(tf_device=tf_device, num_threads=num_threads)
  File "/opt/frigate/frigate/edgetpu.py", line 63, in __init__
    edge_tpu_delegate = load_delegate('libedgetpu.so.1.0', device_config)
  File "/usr/local/lib/python3.8/dist-packages/tflite_runtime/interpreter.py", line 154, in load_delegate
    raise ValueError('Failed to load delegate from {}\n{}'.format(
ValueError: Failed to load delegate from libedgetpu.so.1.0
frigate.mqtt                   INFO    : MQTT connected
frigate.watchdog               INFO    : Detection appears to have stopped. Restarting frigate
frigate.app                    INFO    : Stopping...
frigate.record                 INFO    : Exiting recording maintenance...
frigate.object_processing      INFO    : Exiting object processor...
frigate.events                 INFO    : Exiting event processor...
frigate.events                 INFO    : Exiting event cleanup...
frigate.watchdog               INFO    : Exiting watchdog...`

Anybody could help?

**Comment by hawkeye217** [comment, marked as answer]
You need to specify a CPU detector if you aren’t using a Coral device. It defaults to looking for one.

See the docs here: https://blakeblackshear.github.io/frigate/configuration/detectors

**Reply by McPaHa** [reply]
Yeap I realized that earlier today but haven't time to remove my question. Thx for this!
